### DISCUSSION TITLE
ComboBox with the same `Watermark` as TextBox

### DISCUSSION THREAD

**Discussion by MihailsKuzmins** [AvaloniaUI/Avalonia#9941] [Q&A]
I know that there is this issue https://github.com/AvaloniaUI/Avalonia/issues/4878 about `Watermark` and `Placeholder`, but I was wondering is it possible to create the same `Watermark` effect with `UseFloatingWatermark` for ComboBox? Because `PlaceholderText` is not exactly the same (it disappears when there is a selected value), so I think it would be nice to be able to style ComboBox in the same way.

![image](https://user-images.githubusercontent.com/47413092/211295179-00866b56-17fa-4974-b456-e314e5d55fcf.png)

**Comment by maxkatz6** [comment]
It should be possible, if you change default template of the ComboBox in a way how floating watermark is implemented in the text box.
To be specific, it just shows additional [element](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/TextBox.xaml#L136) on [a condition when text is not empty](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Themes.Fluent/Controls/TextBox.xaml#L229).
